wa/framework/configuration/core.py

```diff
@@ -945,6 +957,8 @@ def add_workload(self, workload):
 
     def disable_augmentations(self, augmentations):
         for entry in augmentations:
+            if entry == '~~':
```

Comment:
This is correct behaviour -- should be specifying `--disable ~` intstead. This is consistent with `~` usage elsewhere -- `augmentations: [~dmesg]` is equivalent to `--disable dmesg` (the leading `~` is dropped); therefore `augmentations: [~~]` is equivalent to `--disable ~`. But yeah, maybe should handle that  specially, as likely to be a common error.

Reply:
Added explicit check for this condition, with a more meaningful error message.

Reply:
On second thought, `~` will need to be escaped in most (all?) shells, and the intent is sufficiently unambiguous, at least, when processing `--disable`  values, that might as well auto-correct it, rather than raising an error.